docs/email/components.html

```diff
@@ -0,0 +1,528 @@
+                </div>
+                <div class="grid--cell" aria-label="notice-message">
+                    <p class="m0">
+                        Using <code class="stacks-code">class="has-markdown"</code> applies styles defined in the <code class="stacks-code">&lt;style&gt;</code> tag. Only <a href="https://www.campaignmonitor.com/css/style-element/style-in-head/">email clients that support <code class="stacks-code">&lt;style&gt;</code> in <code class="stacks-code">&lt;head&gt;</code></a> will render <code class="stacks-code">has-markdown</code> links properly.
+                    </p>
+                </div>
+            </div>
+        </aside>
+    </div>
+    {% header {element: 'h3', text: 'Auto-detected links'} %}
+    <p class="stacks-copy">
+        Some email clients auto-detect certain text strings (like dates, times, and locations) and automatically convert them into hyperlinks. We can't <em>remove</em> the link, but we can make the link <em>appear</em> like the text around it by negating a few CSS styles.
+    </p>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<p style="margin: 0 0 15px 0;">On Thursday we'll be doing on planned maintenance on our New York server, so the parts of the site might be unavailable.</p>
+<p style="margin: 0 0 15px 0;">On <span class="unstyle-auto-detected-links">Thursday</span> we'll be doing on planned maintenance on our <span class="unstyle-auto-detected-links">New York</span> server, so the parts of the site might be unavailable.</p>
+{% endhighlight %}
+        <div class="stacks-preview--example" style="font-family: arial, sans-serif; font-size: 15px; line-height: 19px; color: #54595F; text-align: left;">
+            <p style="margin: 0 0 15px 0;">On <a href="https://stackoverflow.com/" style="color: blue; text-decoration: none; border-bottom: 1px dashed;">Thursday</a> we'll be doing on planned maintenance on our <a href="https://stackoverflow.com/" style="color: blue; text-decoration: none; border-bottom: 1px dashed;">New York</a> server, so the parts of the site might be unavailable.</p>
+            <p style="margin: 0 0 15px 0;">On <a href="https://stackoverflow.com/" style="border-bottom: 0 !important; cursor: default !important; color: inherit !important; text-decoration: none !important; font-size: inherit !important; font-family: inherit !important; font-weight: inherit !important; line-height: inherit !important;">Thursday</a> we'll be doing on planned maintenance on our <a href="https://stackoverflow.com/" style="border-bottom: 0 !important; cursor: default !important; color: inherit !important; text-decoration: none !important; font-size: inherit !important; font-family: inherit !important; font-weight: inherit !important; line-height: inherit !important;">New York</a> server, so the parts of the site might be unavailable.</p>
+        </div>
+        <aside class="s-notice s-notice__info s-notice__outlined mt24" role="status" aria-hidden="false">
```

Comment:
Move notice outside of `div.stacks-preview--example`. Getting a connected box-shadow effect.

![image](https://user-images.githubusercontent.com/1544281/43609679-b1772578-9672-11e8-9789-6d422e88e55d.png)